src/v2/view-builder/views/captcha/CaptchaView.js

```diff
@@ -131,10 +135,11 @@ export default View.extend({
     // the 'data-callback' attribute which the Captcha library uses to invoke the callback.
     window[OktaSignInWidgetOnCaptchaSolvedCallback] = onCaptchaSolved;
 
+    const locale = this.options.settings.get('language');
```

Comment:
nit: can we move this locale computation `const locale = this.options.settings.get('language');`  into `getCaptchaUrl` method itself ?

Reply:
As part of this I'll move the method inside the view because otherwise it won't have the right context for `this`